broker-node/app/Clients/BrokerNode.php

```diff
@@ -41,21 +43,15 @@ private static function initMessenger()
 
     public static function processNewChunk(&$chunk)
     {
-        try {
-            if (self::dataNeedsAttaching($chunk)) {
-                self::buildTransactionData($chunk);
-                self::sendToHookNode($chunk);
-            } else {
-                // move on to the next chunk
-                /*
-                 * TODO: is there anything specific we want to do if it's
-                 * already attached?
-                 */
-            }
-        } catch
-        (\Exception $e) {
-            echo "Caught exception: " . $e->getMessage();
-            // something went wrong during our check, do something about it
```

Comment:
let's stop doing this "try / catch error and do nothing" pattern in the future

Reply:
Logging is usually appropriate.  Not sure if we ever decided on a logger.